hoot-core/hoot-core.pro

```diff
@@ -67,3 +67,6 @@ log4cxx {
   SOURCES -= src/main/cpp/hoot/core/util/LogLog4Cxx.cpp
   HEADERS -= src/main/cpp/hoot/core/util/LogLog4Cxx.h
 }
+
+DISTFILES += \
+    ../sonar-project.properties
```

Comment:
Added the `sonar-project.properties` file to the project so that it is easier to modify in the future.